src/PubTypes.lua

```diff
@@ -111,6 +111,11 @@ export type ChildrenKey = Symbol & {
 	-- name: "Children" (add this when Luau supports singleton types)
 }
 
+-- Denotes referebce instances in an instance or component's property table.
+export type RefKey = Symbol & {
+	-- name: "Ref" (add this when Luau supports singleton types)
```

Comment:
This was inconsistent with the value of `name` you used above. The change I suggested there resolves this,